packages/ombits-client/package.json

```diff
@@ -3,19 +3,33 @@
   "version": "0.0.0",
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
+    "@fortawesome/fontawesome-svg-core": "^1.2.25",
+    "@fortawesome/free-solid-svg-icons": "^5.11.2",
+    "@fortawesome/react-fontawesome": "^0.1.7",
+    "@types/luxon": "^1.21.0",
     "apollo-boost": "^0.4.4",
+    "customize-cra": "^0.9.1",
     "graphql": "^14.5.8",
+    "immer": "^5.0.0",
+    "luxon": "^1.21.3",
+    "navi": "^0.13.6",
+    "normalizr": "^3.4.1",
     "om-ui": "^0.0.0",
     "react": "^16.12.0",
     "react-apollo": "^3.1.3",
+    "react-app-rewired": "^2.1.5",
     "react-dom": "^16.12.0",
+    "react-navi": "^0.13.6",
     "react-scripts": "3.2.0",
+    "react-spring": "^8.0.27",
+    "react-use": "^13.10.0",
+    "styled-jsx": "^3.2.4",
     "typescript": "^3.7.2"
   },
   "scripts": {
-    "dev": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "dev": "react-app-rewired start",
+    "build": "react-app-rewired build",
```

Comment:
@remotealex :sweat_smile: how you come across this

Reply:
It's commonly used with CRA to avoid ejecting